lib/active_merchant/billing/gateways/simetrik.rb

```diff
@@ -127,8 +126,8 @@ def supports_scrubbing?
 
       def scrub(transcript)
         transcript.
-          gsub(%r((\"number\\\":\\\")\d+), '\1[FILTERED]').
-          gsub(%r((\"security_code\\\":\\\")\d+), '\1[FILTERED]').
+          gsub(%r(("number\\?":\\?")\d+), '\1[FILTERED]').
+          gsub(%r(("security_code\\?":\\?")\d+), '\1[FILTERED]').
           gsub(%r((\"exp_month\\\":\\\")\d+), '\1[FILTERED]').
           gsub(%r((\"exp_year\\\":\\\")\d+), '\1[FILTERED]').
           gsub(%r((\"holder_first_name\\\":\\\")"\w+"), '\1[FILTERED]').
```

Comment:
So, one issue I'm having with this Very Important Method is that it claims to scrub `exp_month`, `exp_year`, `holder_first_name`, and `holder_last_name` from the transcript. However, when I examine the scrubbed transcript from the `def test_scrub` unit test, I can see that only the `number`, and `security_code` values have been scrubbed.

I think we need to:
* determine what are the actual sensitive values we should scrub. I think the exp dates and names are not typically scrubbed values, so maybe we can just remove them?
* improve the `def test_scrub` test so that it checks every value that we claim to be scrubbing

Reply:
I took out exp_month, exp_year, holder_first_name, holder_last_name and added Bearer and client_secret. Test has been updated as well.